generators/server/index.js

```diff
@@ -50,7 +50,7 @@ module.exports = generators.Base.extend({
         'cors': '2.8.4',
         'csurf': '1.9.0',
         'debug': '3.1.0',
-        'healthcheck-fastit': 'git+ssh://git@github.com:fastit/health-check.git#1.0.1',
+        'healthcheck-fastit': 'git+ssh://git@github.com/fastit/health-check.git#1.0.1',
```

Comment:
why do you change this ? it works fine with `:`

Reply:
with yarn 0.19.1 I had an non-blocking error during my tests -> https://stackoverflow.com/questions/40016023/yarn-unable-to-clone-private-github-repo-ssh-could-not-resolve-hostname-githu/40630175#40630175